.envrc

```diff
@@ -0,0 +1 @@
+dotenv .env
```

Comment:
Not sure if you meant to include this?

Reply:
It is needed for direnv to work so it loads all the .env variables into the terminal whenever we are in the root folder. Otherwise the configuration.ts is not able to detect `process.env` variables and it breaks locally. I think there is also a way in NestJS to do this via the `ConfigModule` (https://docs.nestjs.com/techniques/configuration). Maybe something we can look into separately cc @PooyaRaki

Reply:
Is this [feature-related](https://github.com/safe-global/safe-client-gateway/blob/main/src/app.module.ts#L65-L67)? If this is a general issue (and not related to the entity in question), this should be done in a separate PR.

Reply:
Its not feature related but it doesn't work locally unless we hard-code the config which is cumbersome. This solves it for whoever is using direnv. The .envrc file was already there but empty. How are you handling .env variables locally to make them work?